drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/AcceptedCommandList.lua

```diff
@@ -0,0 +1,75 @@
+local cluster_base = require "st.matter.cluster_base"
```

Comment:
Do we need this file? Air Quality doesn't have any commands?

Reply:
Removed in 2df4f7a8fa7e0847b68a82f2e06417ac63f957cf!